Updated fiscalizations readme (#42)
MewsSystems/fiscalizations: README.md

# Mews fiscalizations

This repository contains multiple projects. Each project supports reporting of e-invoices to the corresponding country's government authority.

**Please note that we're not responsible for how these libraries are used and if the data provided is correct or not.**
**Also, it is important to note that these libraries were created to meet our usages, so they might be missing some data that can be important in your case (feel free to extend any library and create a PR).**

> P.S Each project folder has it's own documentation (as described below in section **Projects**).

Reviewer: Let's also put this sentence after the first line: `...ntry's government authority.`

```suggestion
Each project folder has it's own documentation. Use the link in the table below to get there.
```

# Mews fiscalizations

> This repository contains a list of projects where each project supports reporting of e-invoices to the corrosponding country's government authority.

Reviewer: ```suggestion
This repository contains multiple projects. Each project supports reporting of e-invoices to the corresponding country's government authority.
```

> For now, we support reporting of e-invoices in the following countries: Austria, Czech Republic, Germany, Spain, Hungary and Italy.

Reviewer: I'd delete this. The list of projects is literally right below 😄

**Please note that we're not responsible for how these libraries are used and if the data provided is correct or not.**
**Also, it is important to note that these libraries were created to meet our usages, so they might be missing some data that can be important in your case (feel free to extend any library and create a PR).**

P.S Each project folder has it's own documentation (as described below in section **Projects**).

## Builds and tests

> | **Project** | **Windows** | **Linux** | **Notes** |

Reviewer: I'm thinking that we could make the first column in this table be a link to the folder (and remove the `Projects` section below)
And then if we add one more column with a link to the nuget package, we could remove the Nuget section too. I'm a bit worried that the columns would be too small, but I'm hoping that the `notes` column would get smaller and all would be good. What do you think? 

Alternatively, we could instead do header for each country here and put the 
- link to the readme
- link to the nuget
- some basic information.

But I like the table more. Because whatever information here is basically duplicit with the readme for that country directly.

@marektresnak opinion on this?

Reviewer 2: I like the table idea as well. Seems more compact and better organized.
